assembl/static2/js/app/components/debate/survey/proposals.jsx

```diff
@@ -43,15 +43,18 @@ class Proposals extends React.Component {
     })}`;
     return (
       <div className="shown">
-        <h3 className="collapsed-title">
-          <span>{questionTitle}</span>
-          <div className={postsToShow.length > 0 ? 'shown proposal-arrow' : 'hidden proposal-arrow'}>
-            <span
-              className={this.state.hideProposals ? 'assembl-icon-down-open color pointer' : 'assembl-icon-up-open color pointer'}
-              onClick={this.displayProposals}
-            />
-          </div>
-        </h3>
+        <div onClick={this.displayProposals} className="pointer">
+          <h3 className="collapsed-title padding-title">
+            <span className="title-txt">{questionTitle}</span>
+            <div className={postsToShow.length > 0 ? 'shown proposal-arrow' : 'hidden proposal-arrow'}>
```

Comment:
You can use `import classnames from 'classnames'` but at this level of complexity, it's not very important